### PR TITLE
Update CTAs to use the new design

### PR DESCRIPTION
**Pull request by usha-mandya**
- Update font for CTAs in the dark-mode to white
- Update existing CTAs to use the new design
- Use specific UTMs for each feature

### CODE REVIEW

**Review by jerae-duffin — COMMENTED** [review]
LGTM!

**Review by jerae-duffin — DISMISSED** [review]
LGTM

File: docker-hub/audit-log.md [hunk 1]
```diff
@@ -4,7 +4,11 @@ keywords: Team, organization, activity, log, audit, activities
 title: Audit log
 ---
 
-> The Audit log feature is available for users subscribed to a Team plan. [Upgrade](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"} your existing account to a Team account to start using the Audit log feature.
+{% include upgrade-cta.html
+  body="The Audit log feature is available for users subscribed to a Team plan. Upgrade your existing account to start using the Audit log feature."
```
Comment (mark-dr) [inline docker-hub/audit-log.md:8]: Users can't be on a Team plan, and this feels slightly repetitive. How about:

```suggestion
  body="The Audit log feature is available for organizations subscribed to a Team plan. Upgrade your existing plan to start tracking events across your organization."
```

File: docker-hub/builds/index.md [hunk 2]
```diff
@@ -10,6 +10,11 @@ redirect_from:
 title: Set up Automated Builds
 ---
 
+{% include upgrade-cta.html
+  body="The Automated Builds feature is available for Pro or Team plans only. Upgrade your existing account to a Pro or a Team plan to start using Automated Builds."
```
Comment (mark-dr) [inline docker-hub/builds/index.md:14]: Slight technical correction, bit less repetition:

```suggestion
  body="The Automated Builds feature is available on Pro and Team plans. Upgrade your existing plan to start automatically building and pushing your images today."
```

File: docker-hub/vulnerability-scanning.md [hunk 3]
```diff
@@ -4,7 +4,11 @@ keywords: scanning, vulnerabilities, Hub, Snyk
 title: Hub Vulnerability Scanning
 ---
 
-> Docker Hub Vulnerability Scanning is available for developers subscribed to a Pro or a Team plan. For more information about the pricing plans, see [Docker Pricing](https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade){: target="_blank" rel="noopener" class="_"}.
+{% include upgrade-cta.html
+  body="Docker Hub Vulnerability Scanning is available for developers subscribed to a Pro or a Team plan. Upgrade your existing account to start using Hub Vulnerability Scanning."
```
Comment (mark-dr) [inline docker-hub/vulnerability-scanning.md:8]: Upgrade _plan_, not account. Reduce repetition.

```suggestion
  body="Docker Hub Vulnerability Scanning is available for developers subscribed to a Pro or a Team plan. Upgrade your existing plan to get automatic vulnerability scans for your images."
```

**Review by mark-dr — CHANGES_REQUESTED** [review]

File: desktop/dashboard.md [hunk 4]
```diff
@@ -229,7 +229,11 @@ To interact with remote repositories:
 
 The **Volumes** view in Docker Dashboard enables you to easily create and delete [volumes](../storage/volumes/index.md) and see which ones are being used. If you are a developer subscribed to a Pro or a Team plan, you can also see which container is using a specific volume and explore the files and folders in your volumes.
 
-> [Upgrade](https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade){: target="_blank" rel="noopener" class="_"} your existing account to a Pro or a Team plan to start exploring the details in your volume.
+{% include upgrade-cta.html
+  body="Upgrade your existing account to a Pro or a Team plan to start exploring the details in your volume."
```
[the remark this answers is not in the document]
Comment (usha-mandya) [reply desktop/dashboard.md:233]: Makes sense. How about making it even simpler "To start exploring the details in your volumes, upgrade to a Pro or a Team plan."

File: docker-hub/builds/index.md (hunk 2 above)
Comment (usha-mandya) [reply docker-hub/builds/index.md]: Updated

File: docker-hub/vulnerability-scanning.md (hunk 3 above)
Comment (usha-mandya) [reply docker-hub/vulnerability-scanning.md]: Updated

File: docker-hub/audit-log.md (hunk 1 above)
Comment (usha-mandya) [reply docker-hub/audit-log.md]: Updated

**Review by mark-dr — APPROVED** [review]

**Review by jerae-duffin — APPROVED** [review]
LGTM

**Pull request merged**